src/_includes/layouts/base.html

```diff
@@ -43,7 +43,7 @@
     <meta property="og:image:alt" content="{{ cover_image_alt }}">
     <meta property="og:locale" content="{{ site.locale | replace("-", "_") }}">
     <meta property="og:type" content="website">
-    <meta property="og:url" content="{{ metadata.url }}{{ page.url }}">
+    <meta property="og:url" content="{{ page_url }}">
```

Comment:
This is always `""` because `page_url` variable was not set.

Reply:
Doh! Got crossed up with the docs base.html